package.json

```diff
@@ -28,6 +28,7 @@
   "dependencies": {
     "espower-source": "^1.0.0",
     "minimatch": "^3.0.0",
+    "typescript": "1.8.0-dev.20151106",
```

Comment:
typescript 1.7+ has public API to convert tsconfig.json to CompilerOption.
TODO: use typescript 1.7 when it is relesed.